packages/table/src/table.tsx

```diff
@@ -615,6 +622,32 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         }
     }
 
+    private syncScrollPosition(_prevProps: ITableProps, prevState: ITableState) {
```

Comment:
Why are you passing prevProps if you don't need it though, then? Consistency, of a sort?